lib/PathSegmentRegistry.php

```diff
@@ -35,6 +35,17 @@ public function __construct(array $pathSegments = [])
     }
 
     /**
+     * Add a pathsegment with given name.
+     *
+     * @param string $name
+     * @param string $pathSegment
+     */
+    public function add($name, $pathSegment)
```

Comment:
If the suggestion in https://github.com/sulu-io/sulu-document-manager/pull/58/files#diff-aaf1b8253014c99a5f92c8b8de4d9515R43 would be implemented, we don't even need this method.